easybuild/tools/toolchain/toolchain.py

```diff
@@ -39,7 +39,7 @@
 from vsc.utils.missing import nub
 
 import easybuild.tools.toolchain
-from easybuild.tools.build_log import EasyBuildError, dry_run_msg
+from easybuild.tools.build_log import EasyBuildError, dry_run_msg 
```

Comment:
space?

Reply:
Run pep8 to find these things?

Reply:
Yeah, I should, but there's a lot of noise there too currently... Fixed, thx